src/components/bottom-sheets/account-management/AccountShellMergeSuccessBottomSheet.tsx

```diff
@@ -151,6 +151,10 @@ const styles = StyleSheet.create( {
     marginTop: 'auto',
     flexDirection: 'row',
     justifyContent: 'flex-start',
+    shadowColor: Colors.shadowBlue,
+    shadowOpacity: 1,
+    shadowOffset: { width: 15, height: 15 },
+    elevation: 5
```

Comment:
@cakesoft-nikhita These styles are being repeated a few times in different components. Is it not possible to use a common style for these components? Is there any reason why they need to be present separately in different components?

Reply:
Made a common component of Button